module/zfs/zcp_get.c

```diff
@@ -230,7 +230,7 @@ get_special_prop(lua_State *state, dsl_dataset_t *ds, const char *dsname,
 	char *strval = kmem_alloc(ZAP_MAXVALUELEN, KM_SLEEP);
 	char setpoint[ZFS_MAX_DATASET_NAME_LEN] =
 	    "Internal error - setpoint not determined";
-	zfs_type_t ds_type;
+	zfs_type_t ds_type = -1;
```

Comment:
Is it guaranteed that `get_objset_type(ds, &ds_type)` won't fail below?  Otherwise, you might want to check the return code.

Reply:
It's void-casted there (and used elsewhere) so I think so, yeah.